### ISSUE TITLE
ch-grow crashes with file not found

### ISSUE THREAD

**Issue by trandles-lanl** [hpc/charliecloud#810]
[Edited by @reidpr, replacing original report with simpler steps to reproduce.]

```
$ cd ~/charliecloud      # source code
$ git checkout v0.16
$ ./autogen.sh && ./configure && make
$ rm -Rf /var/tmp/$USER  # parent of default $CH_STORAGE_DIR
$ bin/ch-build -b ch-grow -t foo -f examples/Dockerfile.centos7 examples
building with: ch-grow
growing: foo
  7 RUN ['/bin/sh', '-c', 'yum -y install epel-release']
Traceback (most recent call last):
[...]
FileNotFoundError: [Errno 2] No such file or directory: '/var/tmp/reidpr/ch-grow/img/foo/etc/resolv.conf'
```

Some systems seem to exhibit the problem and others don't. Some Dockerfiles work (e.g., `test/Dockerfile.00_tiny`); others give different errors (e.g., `examples/Dockerfile.exhaustive` complains about `/var/tmp/reidpr/ch-grow/img/foo/ch/environment` instead of `resolv.conf`).

**Comment by trandles-lanl** [comment]
I edited the above to fix a typo in the Dockerfile. It was a copy-paste error and doesn't affect the bug report.

**Comment by reidpr** [comment]
I'm pretty sure this is an ordering problem: note that `RUN` is the first instruction executed, not `FROM`. I think it's a duplicate of merged PR #761.

**Comment by reidpr** [comment]
If I'm wrong, please re-open.
